canonical-web-and-design/jaas-dashboard
# Subordinate units

## Done

Added row for subordinate units 

## QA

- Sanity check code
- Check model with known subordinates and verify the correct units are nested
